plugins/teststeps/teststeps_test.go

```diff
@@ -6,7 +6,9 @@
 package teststeps
 
 import (
+	"context"
 	"fmt"
+	"github.com/facebookincubator/contest/pkg/statectx"
```

Comment:
move to other imports further below (also check the other files, just noted it here)